docs/modules/ROOT/pages/network-files.adoc

```diff
@@ -0,0 +1,64 @@
+= Network Files
+
+OpenZeppelin Upgrades keep track of all the implementation contracts you have deployed in an `.openzeppelin` folder in the project root, as well as the proxy admin. You will find one file per network there. It is advised that you commit to source control the files for all networks except the development ones (you may see them as `.openzeppelin/unknown-*.json`).
+
+> Note: the format of the files within the `.openzeppelin` folder is not compatible with those of the xref:cli::index.adoc[OpenZeppelin CLI]. If you want to use these plugins for an existing OpenZeppelin CLI project, we will be sharing soon a guide on how to migrate.
+
+
+[[network.json]]
+== `<network_name>.json`
+
+OpenZeppelin Upgrades will generate a file for each of the networks you work on (`ropsten`, `mainnet`, etc). These files share the same structure:
+
+[source,json]
+----
+// .openzeppelin/<network_name>.json
+{
+  "manifestVersion": "3.0",
+  "impls": {
+    "...": {
+      "address": "...",
+      "txHash": "...",
+      "layout": {
+        "storage": [...],
+        "types": {...}
+      }
+    },
+    "...": {
+      "address": "...",
+      "txHash": "...",
+      "layout": {
+        "storage": [...],
+        "types": {...}
+      }
+    }
+  },
+  "admin": {
+    "address": "...",
+    "txHash": "..."
+  }
+}
+----
+
+For every logic contract, besides the deployment address, the following info is also tracked:
```

Comment:
I think we use "implementation" and "logic" kind of inconsistently. I also don't think either of those terms is good.

Marto has suggested we could use the term "version", and talk about "contract versions". In this sentence we would say "For every contract version, ...".

What do you think?

Reply:
Sounds good. Updated in https://github.com/OpenZeppelin/openzeppelin-upgrades/pull/142/commits/a05349dca998d49951c5adc19c20863e5d059223